src/apps/content-editor/src/app/views/ItemList/ItemList.js

```diff
@@ -56,708 +63,629 @@ export default connect((state, props) => {
         state.content[item].meta.ZUID.slice(0, 3) !== "new" // Don't include new items
     )
   };
-})(
-  class ItemList extends Component {
-    _isMounted = false;
-
-    // We use this ref to determine the list dimensions
-    table = React.createRef();
-    list = React.createRef();
-
-    state = {
-      saving: false,
-      loading: false,
-      loadingSecondary: false,
-      // Keep internal item state for handling table filters performantly
-      // avoiding recalculations on component updates
-      items: [],
-      fields: [],
-      height: 1000,
-      width: 1000,
-      sortedBy: null,
-      status:
-        (this.props.filters[this.props.modelZUID] &&
-          this.props.filters[this.props.modelZUID].status) ||
-        "all"
+})(function ItemList(props) {
+  const history = useHistory();
+  const _isMounted = useRef(false);
```

Comment:
What is the purpose of using a ref when later it is just boolean evaluation?

Reply:
Wouldn't `useState` have the same behavior? 

My concern with a ref is weight and possibility of a memory leak. Meaning under the hood I assume react is allocating more resources when generating a ref vs state. Could there be a potential for memory leaks if `ref` references are being maintained. i.e. Could this create detached DOM nodes?  Maybe a non issue as the ref is not being attached to a DOM node. p.s. I don't have any authoritative answers to these theoretical questions, just posing thoughts for consideration. I'm not viewing this as a blocker